Include/cpython/pyhash.h

```diff
@@ -3,7 +3,7 @@
 #endif
 
 /* Prime multiplier used in string and various other hashes. */
-#define _PyHASH_MULTIPLIER 1000003UL  /* 0xf4243 */
+#define PyHASH_MULTIPLIER 1000003UL  /* 0xf4243 */
```

Comment:
Should we keep the legacy _ name around (but undocumented) to smooth the transition for those few projects that use it in their existing extension code?

`#define _PyHASH_MULTIPLIER PyHASH_MULTIPLIER`

Reply:
oh wait, you already did this below. nevermind! :)